comment/managers/comments.py

```diff
@@ -12,8 +12,7 @@ def all_exclude_flagged(self):
         if not allowed_flags or show_flagged:
             return super().get_queryset()
 
-        return super().get_queryset().select_related('flag').annotate(
-            flags=models.F('flag__count')).filter(flags__lte=allowed_flags)
+        return super().get_queryset().exclude(flag__state__exact=2)
```

Comment:
Instead of using raw numbers maybe we may use Enum choices.

Reply:
Yes I will look at this in future code refactoring, I tried to get `Flag.Flagged` but got a circular import.

Reply:
Yes, that is the reason I had used the statues in managers.